docs/upgrade.md

```diff
@@ -86,6 +86,13 @@ process, for example:
     ```
 # Upgrading to v1.(next)
 
+## Time-based cache expiry is now enabled by default
+
+Formerly, entries in the cache were not evicted regardless of whether they were accessed after storing.
+This behavior has now changed. By default entries in the cache are now evicted after 30m of not being accessed. 
+To change the default behavior, go to the `caches` section of the config and change the `expire_caches` and 
+`cache_entry_ttl` flags as necessary. Please note that these flags replace the `expiry_time` flag in the config.  
+
```

Comment:
I don't know why there's this extra section above `v1.53.0` — I would probably bring them down into 1.53 since we haven't released an RC for it yet.